spec/system/custom_sidebar_sections_spec.rb

```diff
@@ -7,9 +7,13 @@
   let(:sidebar) { PageObjects::Components::Sidebar.new }
 
   it "allows the user to create custom section" do
+    visit("/latest")
+
+    expect(sidebar).to have_no_add_section_button
```

Comment:
Tagging on to an existing test so to reduce testing overhead for a small feature